components/divider/style/index.less

```diff
@@ -9,7 +9,6 @@
 
   background: @border-color-split;
 
-  &,  /* for compatiable */
   &-vertical {
     position: relative;
     top: -0.06em;
```

Comment:
May I ask that what is `top: -0.06em` for? A comment here would be helpful. cc @afc163